### PR TITLE
Branch export

### LINKED ISSUE #97: Add export data to csv functionality
CSV?

### PR DESCRIPTION
**Pull request by SampsonYe1999**
Export command is added.

Command Input format: "export"

A csv file named "patientData" will be automatically created under the "RapidTracerData" directory.
Only patient details are exported and stored in the csv file.

Resolves #97

### CODE REVIEW

**Review by chengjunyuan — CHANGES_REQUESTED** [review]
Would be good to delete the failing test cases

File: docs/img.png
Comment (legionlegion) [inline docs/img.png:1]: nice image

File: src/test/java/seedu/address/logic/commands/ExportCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class ExportCommandTest {
```
Comment (legionlegion) [inline src/test/java/seedu/address/logic/commands/ExportCommandTest.java:21]: nice test cases!

File: src/main/java/seedu/address/logic/commands/ExportCommand.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    }
+
+    /**
+     * Reads the JSON file into a tree of JsonNode Objects.
+     *
+     * @param jsonFile The JSON file to be read.
+     * @return A tree of JsonNode Objects.
+     * @throws CommandException If an error occurs while reading the JSON file or mapping it to the CSV schema.
+     */
+    public JsonNode readJsonFile(File jsonFile) throws CommandException {
```
Comment (legionlegion) [inline src/main/java/seedu/address/logic/commands/ExportCommand.java:96]: nice function

File: docs/img_1.png
Comment (legionlegion) [inline docs/img_1.png:1]: i like how you used fake phone numbers to protect my privacy!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -105,6 +105,30 @@ Format: `add n/NAME p/PHONE_NUMBER d/DATE_TIME [a/ADDRESS] [t/TAG]`
   - `today 10am-2pm` (this will create an appointment from 10am to 2pm with today's date)
   - `tdy 10am-2pm` (this will create an appointment from 10am to 2pm with today's date)
 
+<h4 id="add" style="color: #7393B3">
+  Importing patient data: "import"
```
Comment (legionlegion) [inline docs/UserGuide.md:109]: good documentation!

**Comment by SampsonYe1999** [comment]
> Would be good to delete the failing test cases

Thank you for the suggestion. It worked after I removed the failed test cases and it's associated imports!

**Review by chengjunyuan — COMMENTED** [review]
LGTM. Can let either Min Rei or Shuyuan take a look before merging.

**Review by legionlegion — APPROVED** [review]
LGTM

**Pull request merged**